package.json

```diff
@@ -54,10 +53,11 @@
     "ethereumjs-util": "^5.1.1",
     "ethers": "^5.0.32",
     "gh-pages": "^3.1.0",
+    "node-polyfill-webpack-plugin": "^2.0.1",
```

Comment:
Webpack 4 had Node polyfills by default. For Webpack 5 you need to do it via a plugin.